content/en/blog/2023/any-metric-receiver.md

```diff
@@ -0,0 +1,470 @@
+---
+title: Receive any custom metric with the OpenTelemetry Collector
+linkTitle: Any Metric Receiver
+date: 2023-11-28
+author: '[Severin Neumann](https://github.com/svrnm), Cisco'
+# prettier-ignore
+cspell:ignore: openssl servername noout enddate gomod debugexporter otlpexporter otlphttpexporter transformprocessor otlpreceiver carbonreceiver ottl datapoint webserver Helmuth
+---
+
+If you're a frequent user of OpenTelemetry (OTel) or have read the
+[Observability Primer](/docs/concepts/observability-primer/), you'll be well
+aware of how instrumental OpenTelemetry is _to easily troubleshoot and handle
+novel problems (i.e. “unknown unknowns”)_. However, there may be times when you
+need to address the familiar, mundane aspects, such as monitoring system
+metrics, or setting up alerts to draw your attention to potential issues. These
+metrics could range from host metrics like disk usage, server availability, or
+even the expiration date of your SSL certificates.
+
+The beauty of a solution like OpenTelemetry is that it doesn't just help with
+the _"unknown"_, but can also streamline the management of the _"known"_. This
+can be achieved by utilizing any one of the
+[90+ receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver)
+available for the [OpenTelemetry Collector](/docs/collector), such as the
+[Host Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver)
+or the [HTTP Check Receiver](/blog/2023/synthetic-testing/).
```

Comment:
To get faster to the "custom metrics" topic I feel like these two paragraphs could be somehow reduced to one.

Reply:
Yeah, they got a lot longer than planned, let me see what I can do.

Reply:
fixed